testlist/other.dat

```diff
@@ -20,14 +13,16 @@
 -Darwin,pic32mz-starterkit:.*
 -Darwin,sure-pic32mx:.*
 -Darwin,ubw32:.*
+# We do not have a toolchain for avr32 outside of Microchip login wall.
+# The work was never upstreamed to GCC.
 
-/risc-v,CONFIG_RV32IM_TOOLCHAIN_GNU_RVGL
-
+/avr
+-avr32dev1:nsh
+-avr32dev1:ostest
```

Comment:
let keep the order by moving avr to the header

Reply:
done